controllers/networkchaos/partition/types.go

```diff
@@ -215,7 +215,7 @@ func (r *Reconciler) generateSetName(networkchaos *v1alpha1.NetworkChaos, namePo
 		hasher.Write([]byte(nameRest))
 		hashValue := fmt.Sprintf("%x", hasher.Sum(nil))
 
-		ipsetName = namePrefix + "_" + hashValue[0:17] + "_" + namePostFix
+		ipsetName = namePrefix + "_" + hashValue[0:16] + "_" + namePostFix
```

Comment:
Why 17 was changed to 16?

Reply:
just want to keep the length constant, because I changed the suffix from `old` to `_old`

Reply:
How about writing `const` for a summary length, and use `-` to represent it. It's too hard to catch for me.

Reply:
why `-`?

Reply:
Sorry, I mean minus here.

Reply:
I will add a comment at here